pkg/cmd/agent-task/list/list_test.go

```diff
@@ -181,16 +202,28 @@ func Test_listRun(t *testing.T) {
 			cfg.Set("github.com", "oauth_token", "OTOKEN")
 			authCfg := cfg.Authentication()
 
-			ios, _, stdout, _ := iostreams.Test()
+			ios, _, stdout, stderr := iostreams.Test()
 			ios.SetStdoutTTY(tt.tty)
 
+			var br *browser.Stub
+			if tt.web {
+				br = &browser.Stub{}
+			}
+
 			httpClient := &http.Client{Transport: reg}
 			capiClient := capi.NewCAPIClient(httpClient, authCfg)
 			opts := &ListOptions{
-				IO:         ios,
-				Config:     func() (gh.Config, error) { return cfg, nil },
-				Limit:      tt.limit,
-				CapiClient: func() (*capi.CAPIClient, error) { return capiClient, nil },
+				IO:      ios,
+				Config:  func() (gh.Config, error) { return cfg, nil },
+				Limit:   tt.limit,
+				Web:     tt.web,
+				Browser: br,
+				CapiClient: func() (*capi.CAPIClient, error) {
+					if tt.web {
+						panic("CapiClient should not be invoked when --web is set")
```

Comment:
How about this? It'd be a less noisy test log this way.

```suggestion
						require.FailNow(t, "CapiClient should not be invoked when --web is set")
```

Reply:
Nice, I didn't know about that, thanks!

c5f7be9adbb77c75eae7f652a9f3cdbb9354f13c